### DISCUSSION TITLE
Translators

### DISCUSSION THREAD

**Discussion by sarsamurmu** [zixpo/candybar#142] [General]
If you've helped in translating CandyBar dashboard, first of all, thank you.
Please submit your name here so we can add it to the translators' list.

**Comment by walrus543** [comment]
Vividly8487 (previously known as Primokorn) for French. Thanks.

**Comment by moertel** [comment]
I've helped with some of the German translations. :) Using either `@moertel` or Stefanie Grunwald is fine.

**Comment by purplemanta6754** [comment]
I, Gonzalo Cruz (`@purplemanta6754`), have contributed with Spanish translations for the FAQS and the strings 🇪🇸✨

**Comment by warimiya** [comment]
I've (finally!) translated it to Thai, just waiting for the merge. You may credit me as @wwwwwwari or just Wari. 

...Although I'm not sure if you've forgotten about this thread since the few year-old replies above me still aren't in the app yet 😅

So please excuse my ping! @sarsamurmu

**Reply by sarsamurmu** [reply]
Thank you for reminding me, I literally forgot about it 😅

I have updated the list and it will be on the next update

**Comment by olivertzeng** [comment]
I'm on my way to retranslate zh_TW 🇹🇼
You can credit me as @olivertzeng after i've done my translations!

**Comment by olivertzeng** [comment]
@sarsamurmu I've replaced all the machine translations of zh_TW with my translations! I hope you like it! Candybar is super super sleek and I strongly appreciate your work!
would like to be credited by @olivertzeng on GitHub
